service/controller/resource/nodeautorepair/create.go

```diff
@@ -0,0 +1,116 @@
+package nodeautorepair
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	"github.com/giantswarm/badnodedetector/pkg/detector"
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/giantswarm/aws-operator/service/controller/controllercontext"
+	"github.com/giantswarm/aws-operator/service/controller/key"
+)
+
+const (
+	annotationEnableNodeTermination = "node-auto-repair.giantswarm.io"
```

Comment:
maybe this could be put on better place but not sure where it would be best

Reply:
When this is a general feature we want to provide across providers somehow, then https://github.com/giantswarm/apiextensions/tree/master/pkg/annotation would be a good place.

Reply:
I feel like as soon as you put dashes in a sub domain something is off with the structure. An alternative suggestion below. 

```
"node.giantswarm.io/automation": "terminate-not-ready"
```

Reply:
looks like we are using dashes in the annotation quite often in that package

Reply:
will be using https://github.com/giantswarm/apiextensions/blob/master/pkg/annotation/nodeautomation.go#L4